legiti/src/main/java/com/legiti/LegitiClient.kt

```diff
@@ -23,8 +24,28 @@ class LegitiClient : LegitiService {
         this.androidContext = null
     }
 
+
+    @Deprecated(
```

Comment:
ahhh this is perfect!!

Reply:
It's super nice because android Studio will add a strikethrough line in the function call